src/component/Table.astro

```diff
@@ -44,7 +44,11 @@ const { downloadData } = Astro.props;
   tr td:first-child(){
     width: 100px;
   }
-  tr td:nth-child(2){
-    width: 125px;
+  tr td:nth-child(3){
+    width: 330px;
   }
-</style>
\ No newline at end of file
+  .hash {
+   font-weight: 300;
```

Comment:
OK. Maybe someday we'll make them roll-over text or something.

Reply:
Maybe a link to instructions for how to verify based on those hashes? I'll leave that for a future PR.